hack/local-up-chaos-mesh.sh

```diff
@@ -107,7 +107,7 @@ fi
 
 if [ -z "$SKIP_IMAGE_BUILD" ]; then
     echo "info: building docker images"
-    IMAGE_REGISTRY_PREFIX=$IMAGE_REGISTRY_PREFIX IMAGE_PROJECT=chaos-mesh IMAGE_TAG=$IMAGE_TAG UI=1 SWAGGER=1 make image
+    IMAGE_REGISTRY_PREFIX=$IMAGE_REGISTRY_PREFIX IMAGE_TAG=$IMAGE_TAG UI=1 SWAGGER=1 make image
```

Comment:
```suggestion
    IMAGE_TAG=$IMAGE_TAG UI=1 SWAGGER=1 make image
```

If I understand correctly, now we are no longer providing the ability to customize the registry, and some targets with `docker-push` prefix also are being removed.

I agree with these changes, since it's easy to rename the image name with `docker image tag`, we don't need to keep these seemingly convenient, but somewhat redundant targets.

So the `IMAGE_REGISTRY_PREFIX` env should be removed here as well as others.

Reply:
> If I understand correctly, now we are no longer providing the ability to customize the registry, and some targets with `docker-push` prefix also are being removed.

Yeah, that's the thing I want to do: do NOT provide the flags/ENVs for configuring different registry/repo, but **only during the building process**.

It seems the changes in `local-up-chaos-mesh.sh` is NOT a part of building things but installing things. 

So maybe it's better to revert the changes in `local-up-chaos-mesh.sh`.

If necessary,  I would open another PR for similar changes, but for installing scripts like `install.sh`, `local-up-chaos-mesh.sh` and other .sh scripts.

What do you think about it? Reverting changes in `local-up-chaos-mesh.sh`

Reply:
> What do you think about it? Reverting changes in local-up-chaos-mesh.sh

I still don't think it's necessary because these scripts are used more for debugging purposes, and giving them more configurable items is often redundant. As I said above:

_I agree with these changes, since it's easy to rename the image name with `docker image tag`, we don't need to keep these seemingly convenient, but somewhat redundant targets._

Reply:
@STRRL Only this place is not updated, after the update I will approve this PR.

Reply:
Hi @g1eny0ung , I actually prefer to revert changes in `local-up-chaos-mesh.sh` **in this PR**.

I totally agree with your opinion. But it's a little out of the scope from this PR.

I would do it, but in another new PR. Not carry changes in `.sh` files in the PR.

What do you think about it?

Reply:
> But it's a little out of the scope from this PR.

@STRRL That sounds reasonable. I agree that we can wrap up the `.sh` modifications in the upcoming PR.